BREAKING CHANGES(stylelint-config): Switch to a community-maintained configurations (#96)
lmc-eu/code-quality-tools: packages/stylelint-config/README.md

> We assume you are using Prettier. That's why we also recommend adding
> `stylelint-config-prettier` above.

> New configuration extends community maintained config `stylelint-config-standard-scss`.

Reviewer: Can you link those configs with their documentation sites or GitHub, please?

Author: Links added.

> New configuration extends community maintained config [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss).

Reviewer 2: I would suggest avoiding the word _new_. It is new now, but this sentence will outdate soon.

Author: True true, fixed.

This config extends another community maintained configs:

- [stylelint-config-recommended](https://github.com/stylelint/stylelint-config-recommended)
- [stylelint-config-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss)

> - [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard)

Reviewer 2: This made me confused—I was thinking it's about _this_ config, not about `stylelint-config-standard-scss`.

Could you maybe improve it so the structure is clearer? Community configs have it like this:

<img width="783" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/191070782-0ae9917c-2df3-4294-b079-85f8a4f29eaf.png">

Author: Good point, it's been re-texted.

## Configuration

### `@lmc-eu/stylelint-config`

Use this ruleset to configure Stylelint to work with your code.

## Recommended stylelint config

```js
// .stylelintrc.js

'use strict';

module.exports = {
  extends: ['@lmc-eu/stylelint-config', 'stylelint-config-prettier'],
};
```

<details>
<summary><i>.stylelintrc</i> or <i>.stylelintrc.json</i></summary>

```json
{
  "extends": ["@lmc-eu/stylelint-config", "stylelint-config-prettier"]
}
```

</details>

<details>
<summary><i>package.json</i></summary>

```json
{
  "stylelint": {
    "extends": ["@lmc-eu/stylelint-config", "stylelint-config-prettier"]
  }
}
```

</details>

---

## Legacy Configurations

> Use this config to keep less strict ruleset that consists from the original one up to this version:

Reviewer 2: ```suggestion
Use this config to keep less strict ruleset that consists from the original one up to version 4.x:
```

`@lmc-eu/stylelint-config/legacy`

> or its extended version by rules for properties order (formerly known as `optional` until now):

Reviewer 2: Again (IMO):

```suggestion
or its extended version with rules for properties order (formerly known as `optional` until now):
```

or

```suggestion
or its version extended by rules for properties order (formerly known as `optional` until now):
```

Author: Thank you, it was fixed.

> or its extended version by rules for properties order (formerly known as `optional` until now):

Reviewer 2: And again the version number to make it future proof:

```suggestion
or its extended version by rules for properties order (formerly known as `optional` until version 5.x):
```

Author: Fixed but this you may be misunderstood - this line still explains old config but also extended by properties order. This line should still point to 4.0.1 version, the old one.
